examples/intro-script-with-gain.xml

```diff
@@ -0,0 +1,11 @@
+...
+    <div begin="25s" end="28s">
+      <p>
+        <animate begin="0.0s" end="0.3s" tta:gain="1;0.39" fill="freeze"/>
+        <animate begin="2.7s" end="3s" tta:gain="0.39;1"/>
+        <span begin="0.3s" end="2.7s">
```

Comment:
It's interesting that the timing is added to the span and not to the audio. This means that the text timing is modified when adding audio mixing instructions. I had not realized it.

Reply:
Not sure I understand?